src/main/kotlin/no/nav/syfo/services/JournalService.kt

```diff
@@ -50,19 +51,28 @@ class JournalService(
                 receivedDialogmelding.personNrPasient,
                 vedleggListe
             )
-            val journalpost = dokArkivClient.createJournalpost(journalpostPayload, loggingMeta)
-
+            val journalpost = try {
+                dokArkivClient.createJournalpost(journalpostPayload, loggingMeta)
+            } catch (exc: Exception) {
+                if (jpRetryEnabled) {
+                    throw exc
+                } else {
+                    logger.warn("Journalføring failed, skipping retry: ", exc)
+                }
+                null
+            }
+            val journalpostId = journalpost?.journalpostId ?: "0"
```

Comment:
Betyr dette at vi lagrer dialogmelding med journalpostId satt til 0 i db i dev? Hadde det vært bedre og gjort dette i de andre appene i dev også (i stedet for å skru av journalføring). Det gjør det kanskje enklere å teste journalføringen i dev :thinking:

Reply:
Ja, det er det jeg endte opp med å gjøre her siden journalføringen er et "trinn" på veien mot å ferdigbehandle. Ja, jeg funderte om vi kunne gjort det på denne måten i de andre app'ene også.. Er ikke helt sikker.

Reply:
Da vil vi i hvert fall kunne teste journalføring (og nedstrøms-greier som er avhengig av at journalpostId er satt) uten å måtte enable cronjob og prosessere feilende journalføringer. Kunne i så fall laget en egen feilhåndtering av manglende aktør-id feilen som bare er aktiv i dev og som setter journalpostId til 0.

Reply:
Ja, gjør en tilpasning for dette i de andre PR'ene. Men denne er kanskje ok da?